Taskfile.yml

```diff
@@ -340,3 +340,20 @@ tasks:
         sh: cat ../internal/util/version/gen/version.txt
       GOARCH:
         sh: go env GOARCH
+
+  security-scan-fs:
+    cmds:
+      - docker run --rm -v $(pwd):/app -w /app aquasec/trivy:latest fs . -q --exit-code=1
```

Comment:
That one might be useful, let's keep it. But let's use the same dependency mechanism we use for other Docker images; see https://github.com/FerretDB/FerretDB/tree/main/build/deps